### PR TITLE
Eugene/feature/about template

### PR DESCRIPTION
**Pull request by JackScher**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,31 +36,42 @@ RUN apt-get update --yes --quiet && apt-get install --yes --quiet --no-install-r
...
+ENV PATH="${PATH}:/root/.poetry/bin"
+COPY poetry.lock .
+COPY pyproject.toml .
+
+RUN POETRY_VIRTUALENVS_CREATE=false poetry install --only main --no-interaction --no-ansi
+RUN pip install python-decouple
+RUN pip install gunicorn
+
+# RUN pip install wagtail-metadata-mixin
+# RUN pip install wagtail
```
Comment (kipod) [inline Dockerfile]: please remove these commented lines

File: app/adapters.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (kipod) [inline app/adapters.py]: Pls divide these this module as separate python modules:
adapters/
  - __init__.py
  - custom_adapter.py
  - social_account_adapter.py

File: app/blocks.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+from wagtail.images.blocks import ImageChooserBlock
```
Comment (kipod) [inline app/blocks.py]: please refactor - each class in a separate python module

File: app/forms.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+class CustomUserCreationForm(UserCreationForm):
+    first_name = forms.CharField(required=True, label=_("First name"))
+    last_name = forms.CharField(required=True, label=_("Last name"))
+    age = forms.CharField(required=True, label=_("Age"))
+    school_class = forms.ModelChoiceField(queryset=SchoolClassPage.objects.all(), required=True, label=_("Your class"))
+
+    class Meta(UserCreationForm.Meta):
+        model = User
+        exclude = ('is_superuser',)
+        fields = ('username', 'email','first_name','last_name', 'school_class', 'age')
```
Comment (kipod) [inline app/forms.py]: please use ```black``` as formatter.
Setup it ```formatting on Save```

File: app/models.py [hunk 5]
```diff
@@ -1,23 +1,24 @@
...
 
 from wagtail.models import Page, Collection
 from wagtail.admin.panels import FieldPanel
-from modelcluster.fields import ParentalKey
+from modelcluster.fields import ParentalKey, ParentalManyToManyField
+from wagtail.fields import StreamField
+
+
+from app.blocks import BaseStreamBlock
+from core.models import ExtPage
```
Comment (kipod) [inline app/models.py]: please to separate

File: app/models.py [hunk 6]
```diff
@@ -33,26 +34,183 @@ class ImagesGallaryPage(Page):
...
+        FieldPanel("role"),
+        FieldPanel("school_class"),
+    ]
+
+
+class SchedulePage(Page): pass
+    # content_panels = Page.content_panels + [
+    #     InlinePanel("lessons")
+    # ]
+
```
Comment (kipod) [inline app/models.py]: It looks like useless code... Isn't it?

File: app/models.py [hunk 7]
```diff
@@ -33,26 +34,183 @@ class ImagesGallaryPage(Page):
...
+
+class LessonPage(Page):
+    time = models.CharField(max_length=5)   #add validatin like "\d{2}:\d{2}"
```
Comment (kipod) [inline app/models.py]: use ``` #TODO ``` style comments

File: app/services [hunk 8]
```diff
@@ -0,0 +1,10 @@
+from django.core.mail import send_mail
+
+
+def send_email(self):
+    url = "http://127.0.0.1:8000/confirm-email/"
+    subject = "email confirmation"
+    user_to = ""
+    user_from = ""
+    message = f"You're receiving this e-mail because user {user_to} has given your e-mail address to register an account on example.com. To confirm email, go to {url}. Thank you!"
+    send_mail(subject, message, user_from, [user_to])
```
Comment (kipod) [inline app/services:10]: are you really need this file?

File: app/utils.py [hunk 9]
```diff
@@ -0,0 +1,9 @@
+from app.models import HomePage, Profile, User
+
+
+def create_profile(user: User) -> None:
+    p = Profile(name=user.first_name, surname=user.last_name, age=user.age, role='student', school_class=user.school_class, 
+                        title=' '.join((user.last_name, user.first_name)), slug=user.pk)
```
Comment (kipod) [inline app/utils.py]: use ```black``` as code formatter

File: app/views.py [hunk 10]
```diff
@@ -1,3 +1,31 @@
 from django.shortcuts import render
+from allauth.account.views import SignupView
+from app.forms import CustomUserCreationForm
+from django.core.mail import send_mail
+# from django.contrib.auth.views import LoginView
```
Comment (kipod) [inline app/views.py]: If you don't need this code, you don't need comment-out it - simply remove it. All changes history in Git!

File: app/views.py [hunk 11]
```diff
@@ -1,3 +1,31 @@
...
+        # print(form['email'].value())
+        # print('====================================================================================')
+
+        # send_mail('This is the title of the email',
+        #         'This is the message you want to send',
+        #         'dzhek.layt298@gmail.com',
+        #         [
+        #             form['email'].value(), # add more emails to this list of you want to
+        #         ]
+        # )
```
Comment (kipod) [inline app/views.py]: pls remove old code

File: app/views.py [hunk 12]
```diff
@@ -1,3 +1,31 @@
...
+        return super().post(request, *args, **kwargs)
+
+
+
+# class LoginUserView(LoginView):
+#     form_class = LoginUserForm
+#     template_name = "auth_smth/login_page.html"
```
Comment (kipod) [inline app/views.py]: flake8 must get the warning if the file has not EOL in the EOF

File: core/models.py [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+
+class ExtPage(MetadataPageMixin, Page):
+    # search_image = models.ForeignKey(
+    #     get_image_model_string(),
+    #     verbose_name=_('search image'),
+    #     null=True,
+    #     blank=True,
+    #     on_delete=models.SET_NULL,
+    #     related_name='+',
+    # )
```
Comment (kipod) [inline core/models.py]: old code ?

File: sample.pyproject.toml [hunk 14]
```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
```
Comment (kipod) [inline sample.pyproject.toml]: are you really need this file?

File: start_server.sh [hunk 15]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+echo version
+python manage.py --version
```
Comment (kipod) [inline start_server.sh]: please remove test code

File: templates/base.html [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+                    <div class="footer__image">
+                        <img src="https://cdn.dribbble.com/userupload/3158902/file/original-7c71bfa677e61dea61bc2acd59158d32.jpg?resize=400x0" alt="">
+                    </div>
+                    <div class="footer__menu">
+                        <a href="{% slugurl 'core' %}">Home page</a>
+                        <a href="{% slugurl 'about-us' %}">About us</a>
+                        <!-- <a>Text 3</a>
+                        <a>Text 4</a>
+                        <a>Text 5</a>
+                        <a>Text 6</a> -->
```
Comment (kipod) [inline templates/base.html]: pls remove old code

**Review by kipod — COMMENTED** [review]
Thank you!
Please use ```black``` and ```flake8```
and also in the future when you declare a new python class - define it in a separate module.

Greate job!

**Review by Xavries — APPROVED** [review]

**Pull request merged**